### PR TITLE
[Rule Tuning] 3 tunings to reduce FPs

### PR DESCRIPTION
**Pull request by Aegrah**
## Summary
Just 2 small tunings to reduce FPs on these rules back to 0.

**edit**
I added one more tuning to address issue [#3041](https://github.com/elastic/detection-rules/issues/3041). This will reduce FPs and trigger on TPs:

image

### CODE REVIEW

File: rules/linux/execution_abnormal_process_id_file_created.toml [hunk 1]
```diff
@@ -74,19 +74,20 @@ references = [
 risk_score = 47
 rule_id = "cac91072-d165-11ec-a764-f661ea17fbce"
 severity = "medium"
-tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Execution", "Threat: BPFDoor", "Resources: Investigation Guide", "Data Source: Elastic Endgame"]
+tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Execution", "Threat: BPFDoor", "Resources: Investigation Guide", "Data Source: Elastic Defend"]
 timestamp_override = "event.ingested"
 type = "new_terms"
 
 query = '''
 host.os.type : "linux" and event.category : "file" and event.action : ("creation" or "file_create_event") and
```
Comment (w0rk3r) [inline rules/linux/execution_abnormal_process_id_file_created.toml:82]:
```suggestion
host.os.type : "linux" and event.category : "file" and event.action == "creation" and
```

**Review by w0rk3r — APPROVED** [review]

File: rules/linux/execution_abnormal_process_id_file_created.toml (hunk 1 above)
Comment (Aegrah) [reply rules/linux/execution_abnormal_process_id_file_created.toml:82]: Good catch, but it's KQL rather than EQL. Will fix it! Thanks

Comment (Aegrah) [inline rules/linux/execution_abnormal_process_id_file_created.toml:82]:
```suggestion
host.os.type:linux and event.category:file and event.action:creation and
```

File: rules/linux/execution_abnormal_process_id_file_created.toml [hunk 2]
```diff
@@ -74,19 +74,20 @@ references = [
...
-not process.name : ("runc" or "ufw" or "snapd" or "snap" or "iptables" or "libvirtd" or "containerd-shim-runc-v2" or 
-"ifdown" or "snap-confine" or "ifup" or "dhclient" or "containerd")
+user.id : "0" and file.extension : ("pid" or "lock" or "reboot") and file.path : (/var/run/* or /run/*) and (
+  (process.name : (
+    bash or dash or sh or tcsh or csh or zsh or ksh or fish or ash or touch or nano or vim or vi or editor or mv or cp)
+  ) or (
+  process.executable : (
+    ./* or /tmp/* or /var/tmp/* or /dev/shm/* or /var/run/* or /boot/* or /srv/* or /run/*
+  ))
+) and not process.name : (go or git)
```
Comment (Aegrah) [inline rules/linux/execution_abnormal_process_id_file_created.toml:90]:
```suggestion
user.id:0 and file.extension:(pid or lock or reboot) and file.path:(/var/run/* or /run/*) and (
  (process.name : (
    bash or dash or sh or tcsh or csh or zsh or ksh or fish or ash or touch or nano or vim or vi or editor or mv or cp)
  ) or (
  process.executable : (
    ./* or /tmp/* or /var/tmp/* or /dev/shm/* or /var/run/* or /boot/* or /srv/* or /run/*
  ))
) and not process.name : (go or git)
```

File: rules/linux/execution_abnormal_process_id_file_created.toml (hunk 1 above)
Comment (w0rk3r) [reply rules/linux/execution_abnormal_process_id_file_created.toml]: >but it's KQL rather than EQL

Oops

**Review by Samirbous — APPROVED** [review]

**Pull request merged**